src/lib.rs

```diff
@@ -14,22 +14,22 @@ pub fn clockboard(
     polygons
 }
 
-fn makecircle(centerpoint: Point<f64>, radius: f64) -> Polygon<f64> {
```

Comment:
For future reference, my advice for this sort of thing would be to keep `makecircles` returning a `Polygon` and internally generating `Point`s. These are types with more semantics beyond "a list of 2 numbers". You can then add on another method to export the results in a matrix form, but it ought to take in any `geo::Polygon<f64>` and use https://docs.rs/geo/0.17.1/geo/struct.Polygon.html#method.exterior or similar to ask for the points. Use types in as much as your code as possible; only deal with raw matrices as input/output when necessary.

Reply:
:+1: